production/scrapers/north_carolina_vaccine.R

```diff
@@ -2,49 +2,51 @@ source("./R/generic_scraper.R")
 source("./R/utilities.R")
 
 north_carolina_vaccine_pull <- function(x){
-    "1VhAAbzipvheVRG0UWKMLT6mCVQRMdV98lUUkk-PCYtQ" %>%
-        googlesheets4::read_sheet(sheet = "NC Vaccine", 
-                                  col_types = "Dccccc")
-}
-
-north_carolina_vaccine_restruct <- function(x){
     x %>%
-        filter(!is.na(Date)) %>% 
-        filter(Date == max(Date))
+        xml2::read_html()
 }
 
-north_carolina_vaccine_extract <- function(x, exp_date = Sys.Date()){
-    
-    error_on_date(first(x$Date), exp_date)
+north_carolina_vaccine_restruct <- function(x){
+    txt <- x %>%
+        rvest::html_nodes("section") %>%
+        rvest::html_text() %>% 
+        .[which(str_detect(., "COVID-19 Vaccination Info"))] %>% 
+        str_squish() %>% 
+        str_split("\n") %>% 
+        unlist() 
```

Comment:
What do you think about adding one line after unlist() to make all the text here lowercase, then editing the matching text patterns in the regex step below to be all lowercase too?

Reply:
That's a good idea, done with the latest commit!